src/test/java/no/unit/nva/model/instancetypes/JournalLetterTest.java

```diff
@@ -0,0 +1,59 @@
+package no.unit.nva.model.instancetypes;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.Range;
+import nva.commons.utils.IoUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class JournalLetterTest {
+
+    public static final String JOURNAL_LETTER_WITH_PEER_REVIEW_TRUE_JSON = "journal_letter.json";
+
+    @DisplayName("Journal letters to editor can be created from JSON")
+    @Test
+    void journalLetterReturnsObjectWhenJsonInputIsCorrectlySerialized() throws JsonProcessingException,
+            InvalidPageTypeException {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Not important here, but usage of JsonUtils.objectMapper will always ensure that we serialize things the same way across the whole project. You get also the bonus that if you make it static you save some lines